### DISCUSSION TITLE
imagens não aparecem no github pages.

### DISCUSSION THREAD

**Discussion by kevinsrm** [frontendbr/forum#2493] [Perguntas e Respostas] [answered]
as imagens que coloquei no site não carregam o que poderia ser a causa?
Link do diretorio: https://github.com/kevinsrm/kevinsrm.github.io
link do site:  https://kevinsrm.github.io/

**Comment by Yures1lva** [comment]
Mano pesquisa algo sobre endereço  correto dos path para github pages, meu site aconteceu esse erro apgo relacionado a isso resolveu

**Comment by vitebo** [comment, marked as answer]
você ta tentando pegar as imagens de dentro da pasta `assets`:

<img width="726" alt="image" src="https://github.com/frontendbr/forum/assets/19560693/d685cbc6-e8a7-4afd-9522-d80df639ef7d">

Acontece que as imagens estão disponíveis na raiz do seu repositório, não na pasta `assets`

[Esse seu commit](https://github.com/kevinsrm/kevinsrm.github.io/commit/974d6da22dd628ba0d1e22d09a28fd6a9250dc1b) provavelmente resolve esse problema, precisa só disparar deploy do github pages novamente.

Mas é legal fazer uma alteração nesse commit, `href="/vingadores.jpg"` vai funcionar melhor do que `href="./vingadores.jpg"`
